app/javascript/components/profile/relations.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div class="profile-relations">
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 2 spaces but found 4 spaces                     vue/html-indent